packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/ContentTypeCollapse/CollapsePropertyMatrix/CarretIcon/index.js

```diff
@@ -6,9 +6,9 @@ const CarretIcon = styled(CarretDown)`
   width: ${10 / 16}rem;
   transform: rotate(${({ $isActive }) => ($isActive ? '180' : '0')}deg);
   margin-left: ${({ theme }) => theme.spaces[2]};
-  * {
+  /* * {
     fill: ${({ theme }) => theme.colors.primary600};
-  }
+  } */
```

Comment:
Don't we need this svg color modification?

Reply:
I forgot to remove the comment but you can test it the Carret color is working on my side with changes made in utils/activeStyle.js!